src/plugins/release-issues/index.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @fileoverview Creates a new issue with the "release" label if the old one is closed
+ * @author Teddy Katz
+ */
+
+const moment = require("moment");
+
+const LABEL_NAME = "release";
+
+const ISSUE_TITLE_FORMAT = "[Scheduled release for ]MMMM Do, YYYY";
+
+const getIssueBody = (releaseDate) => `
+
+The scheduled release on ${releaseDate.format("dddd, MMMM Do, YYYY")} is assigned to:
+
+* (needs volunteers)
+* (needs volunteers)
+
+Please use this issue to document how the release went, any problems during the release, and anything the team might want to know about the release process. This issue should be closed after all patch releases have been completed (or there was no patch release needed).
+
+Resources:
+
+* [Release guidelines](https://eslint.org/docs/maintainer-guide/releases)
+
+`.trim();
+
+module.exports = (robot) => {
+    robot.on("issues.closed", async (context) => {
```

Comment:
Just to keep the api layer clean, I would recommend creating a function and then just referencing that function here.
Based on other plugins, its very easy to see what all events they listen to and also adding stuff later becomes more cleaner.